client/src/components/Editor.jsx

```diff
@@ -147,12 +147,37 @@ function Editor() {
                     alt="Signature"
                   />
                 )}
-              <Typography variant="h5" sx={styles.signatureLine}>
+              <Box sx={styles.signatureAndDateLine} />
+              {state.minutes.signatures.length > 0 &&
+                !!state.minutes.signatures[0].signer && (
```

Comment:
!! might be redundant. If necessary, use Boolean(value). You can read more here about that: https://www.ctomczyk.pl/stop-using-logical-not-and-casting-using-double-exclamation-mark-in-javascript-typescript/914/